src/main/java/org/jenkinsci/plugins/durabletask/WindowsBatchScript.java

```diff
@@ -81,6 +82,10 @@ public String getScript() {
         */
         ps.readStdout().readStderr(); // TODO see BourneShellScript
         ps.start();
+        Proc p = ps.start();
```

Comment:
Looks like a typo but `ps.start()` is called twice here

Reply:
whoops, thanks, fixed in 171c622